projects/packages/publicize/src/class-publicize-setup.php

```diff
@@ -27,6 +27,10 @@ public static function on_plugins_loaded() {
 			$publicize_ui = new Publicize_UI();
 
 		}
+		// Adding on a higher priority to make sure we're the first field registered.
+		// The priority parameter can be removed once we deprecate WPCOM_REST_API_V2_Post_Publicize_Connections_Field
+		add_action( 'rest_api_init', array( new Connections_Post_Field(), 'register_fields' ), 5 );
```

Comment:
Does this mean we should remove the priority? Or is that one on the wpcom side?

Reply:
`WPCOM_REST_API_V2_Post_Publicize_Connections_Field` is on WPCOM as well so we can't remove it until WPCOM is using the Publicize package too. For now we want to get in before it.